File: test/easy/inches-to-feet.test.ts
import { describe, test, expect } from "bun:test";
import { inchesToFeet } from "~/easy/inches-to-feet";
describe("Inches to feet", () => {
    test.each([
        [12, 1],
        [360, 30],
        [3612, 301],
        [324, 27],
        [3012, 251],
        [11, 0],
    ])("%d inches is %d feet", (inches, output) => {
        const result = inchesToFeet(inches);
        expect(result).toStrictEqual(output);
    });
});

File: test/veryeasyswift/InchesToFeetTest.swift
import Testing

@testable import very_easy

@Test(arguments: [(12, 1), (360, 30), (3612, 301), (324, 27), (3012, 251), (11, 0)])
func inchesToFeet(inches: Int, outcome: Int) { #expect(inchesToFeet(inches) == outcome) }
